### DISCUSSION TITLE
Bitwarden issue.  Not connecting to self hosted server

### DISCUSSION THREAD

**Discussion by Josiahjcarter** [dani-garcia/vaultwarden#3802] [Q&A]
I keep getting network error when attempting to fetch resource whenever I sign into bitwarden app on my iPhone or trying to use my Bitwarden plugin on my PC.  I can access all my passwords stored, but cant add anything new.  Please help.

**Comment by stefan0xC** [comment]
What error messages do you get excactly? Can you check

- Is Vaultwarden still running? I.e. can you open the web-vault? Maybe your HTTPS certificate has expired?
-  Is it still reachable from your clients? Maybe you changed your firewall settings (on your server or router)?
